bugsnag-plugin-react-native/src/main/java/com/bugsnag/android/BugsnagReactNativePlugin.kt

```diff
@@ -39,10 +46,12 @@ class BugsnagReactNativePlugin : Plugin {
     override fun unload() {}
 
     @Suppress("unused")
-    fun configure(jsVersion: String): Map<String, Any?> {
+    fun configure(env: Map<String, Any?>?): Map<String, Any?> {
```

Comment:
How can we best keep this interface maintained/documented? It feels a bit unstructured at the moment.

I'd like to minimise the number of top-level keys, so how about making "runtimeVersions" the key to a map of String->String runtime version elements? That way there are fewer magic strings and also the JS layer is in control of what it adds.

Reply:
Using a map value would add additional complexity to the deserializer which I'd prefer to avoid at this stage given there will only be 3 values present, and it's known that this solution works.

It's also worth noting that we can change the structure of values in the map without making a breaking change, so could revisit if we do start sending more values over the bridge in future.